docs/source/index.rst

```diff
@@ -34,4 +33,13 @@
 
    metrics/ctc
    metrics/track_overlap
-   metrics/divisions
\ No newline at end of file
+   metrics/divisions
+
+.. toctree::
+   :maxdepth: 2
+   :hidden:
+   :caption: Test Cases
```

Comment:
I think it would be nice to have a landing page for the test cases that has the generic explanation, and links to the three notebooks. But it's not essential

Reply:
Agreed. I added something but the text could probably use some help 😬